ChangeLog

```diff
@@ -52,6 +48,16 @@ Bugfix
    * Fix double initialization of ECC hardware that made some accelerators
      hang.
 
+API Changes
```

Comment:
Why was the order of `API Changes` compared to `New deprecations` changed?

Reply:
The ChangeLog was automerged by Gilles's script, which has a definite opinion about section ordering.

Reply:
It might have a strong opinion, but it's contrary to the defined order on Confluence, which goes Security -> Features -> API Changes -> New deprecations.

That's the kind of thing I often fix before final release, so maybe best to fix the scripts.
Can you please correct it?